### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by menyagah**
# In this section we:
- Created a bubble sort algorithm that sorts items in an array in ascending order. 
- Created another method bubble_sort_by that sorts the array in both the ascending and descending order using the yield and block method.

# Made Tse requested changes
- Changed error handling statement.

### CODE REVIEW

File: bubblesort.rb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  (arr.length - 1).times do |n|
+    (1...arr.length - n).each do |i|
+      return p 'Please add a block!!' unless block_given?
+
+      orient = yield arr[i - 1], arr[i]
+      arr[i], arr[i - 1] = arr[i - 1], arr[i] if orient.positive?
+    end
+  end
+  arr
+end
```
Comment (MarioDena) [inline bubblesort.rb]: Hello Great job on your project everything is working great except for 1 small error checking! 👍 

## Requested Changes 

- [ ] Your bubble sort by method should return an error if no block is given.

That's all, so a small error handling change and you are good to go!

Cheers!

**Review by MarioDena — CHANGES_REQUESTED** [review]
## Changes required

See my comment! 😄

**Comment by lekegitrepo** [comment]
#### Hi @menyagah and @mugberto :clap: :clap:
#### Good job for implementing the required changes :muscle:   This project is approved.

### _Reviewed by Temitayo Adeleke._

**Review by lekegitrepo — APPROVED** [review]
#### Status: Approved

**Pull request merged**